sources/install/package_wordlists.sh

```diff
@@ -106,6 +105,52 @@ function install_genusernames() {
     add-to-list "genusernames,https://gitlab.com/-/snippets/2480505/raw/main/bash,GenUsername is a Python tool for generating a list of usernames based on a name or email address."
 }
 
+
+
+function install_onelistforall() {
+    # CODE-CHECK-WHITELIST=add-aliases,add-history
+    colorecho "Installing onelistforall"
+    wget https://raw.githubusercontent.com/six2dez/OneListForAll/main/onelistforallmicro.txt -P /opt/lists/
+    wget https://raw.githubusercontent.com/six2dez/OneListForAll/main/onelistforallshort.txt -P /opt/lists/
+    add-test-command "[[ -f '/opt/lists/onelistforallshort.txt' ]]"
+    add-to-list "onelistforall,https://github.com/six2dez/OneListForAll,Rockyou for web fuzzing"
+}
+
+function install_rules(){
+    colorecho "Add rules"
```

Comment:
You need to add CODE-CHECK-WHITELIST=add-aliases,add-history

Reply:
ok. It's add.